include/sentry.h

```diff
@@ -925,6 +925,13 @@ SENTRY_API int sentry_init(sentry_options_t *options);
  */
 SENTRY_API int sentry_shutdown(void);
 
+/**
+ * This will lazily load and cache a list of all the loaded libraries.
```

Comment:
Probably makes sense to document that the value returned is frozen.

Reply:
@Swatinem _nit_: Not sure if "frozen" is clear for everyone. Maybe add a few words to explain that the list should not be modified using any of the `sentry_value_*` functions?